src/views/Dashboard.vue

```diff
@@ -65,7 +65,7 @@ import NcEmptyContent from '@nextcloud/vue/dist/Components/NcEmptyContent.js'
 
 import ConversationIcon from './../components/ConversationIcon.vue'
 
-import { CONVERSATION } from '../constants.js'
+import conversation from '../mixins/conversation.js'
```

Comment:
Dashboard entrypoint will be removed with API v2 (see PR from Raimund), so I'd not change anything here, as it will be lost